includes/core/classes/class-assets.php

```diff
@@ -121,6 +121,15 @@ public function add_global_object(): void {
 	 */
 	public function enqueue_scripts(): void {
 		wp_enqueue_style( 'dashicons' );
+		
+		if(has_block('gatherpress/venue')) {
+			wp_enqueue_style(
+				'gatherpress-leaflet',
+				'https://unpkg.com/leaflet@1.9.4/dist/leaflet.css',
```

Comment:
We'll need to call this locally.

Reply:
@jmarx resolved this issue - thanks Jeff!